client/src/pages/LandingPage/LandingPage.js~HEAD

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import NdiLogoUrl from '../../images/ndiLogo.png';
+import profileUrl from '../../images/profile.png';
+import { ModuleCard } from '../../components/ModuleCard';
+import '../../commonStyles.scss';
+import './styles.scss';
+import { config } from '../../settings';
+import { connect } from 'react-redux';
+import { fetchGameData, fetchScores } from './actions';
+import PropTypes from 'prop-types';
+class LandingPage extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {};
+	}
+
+	//Fetch complete game data.
+	componentWillMount() {
+		fetch(config.baseUrl + '/api/game')
```

Comment:
you can make a reusable function for calling APIs instead of using fetch everywhere